views/index.html

```diff
@@ -1,54 +1,47 @@
-	<div id='divBody'>
-		<img src='logo.png' height='60' /><br />
-		<a href='/settings'>Go To Settings Page</a><br /><br />
-		<a href='/producer'>Go To Producer Page</a><br /><br />
-		<a href='/tally'>Go To Tally Web Viewer Page</a><br /><br />
-		Tally Arbiter was written by Joseph Adams and is distributed under the MIT License.
-		<br /><br />
-		<div id='divLicense'>
-			Copyright 2020 Joseph Adams.<br /><br />
-			Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the 'Software'), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-				<br /><br />
-				The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-				<br /><br />
-				THE SOFTWARE IS PROVIDED 'AS IS', WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
-				</div><br /><br />
-				It is not sold, authorized, or associated with any other company or product.<br /><br />
-				To contact the author or for more information, please visit <a id='bloglink' href='http://www.techministry.blog'>www.techministry.blog</a>.
-				<br /><br />
-	</div>
-</div>
+  <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+    <div class="container">
+      <a class="navbar-brand" href="/"><img src="lib/img/logo_white.png" height="30" alt="" loading="lazy"></a>
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="navbar-toggler-icon"></span> </button>
+      <div class="collapse navbar-collapse" id="navbarNav">
+        <ul class="navbar-nav">
+          <li class="nav-item"> <a class="nav-link active" href="/">Home</a> </li>
+          <li class="nav-item"> <a class="nav-link" href="/tally">Tally View</a> </li>
+          <li class="nav-item"> <a class="nav-link" href="/producer">Producer</a> </li>
+          <li class="nav-item"> <a class="nav-link" href="/settings">Settings</a> </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+  <div class='container'>
+    <div class="jumbotron">
+      <h1 class="display-4">Welcome!</h1>
+      <p class="lead">Tally Arbiter was written by Joseph Adams and is distributed under the MIT License.</p>
+      <div class="tally-qrcode" id="qrcodeRegion"></div>
```

Comment:
Did you write this portion yourself or copy it from the atem-tally-lite project? I'm not convinced it is needed.

Reply:
(regarding the QR code)

Reply:
Idea and some of the code indeed came from the atem-tally-lite project. See my use cases below:
- Typing a IP is time consuming and error heavy, scanning a QR is not
- Sharing a QR is faster & easier than typing a IP, so production wise it will be a little bit faster. (and time is money) 😄

Reply:
Thumbs up on the QR code idea. Sharing the full URL with the camera crew is cumbersome.

Reply:
Makes sense. I am ok with it now.